src/Workspaces/Core/Portable/Workspace/WorkspaceRegistration.cs

```diff
@@ -14,9 +16,9 @@ internal WorkspaceRegistration()
         {
         }
 
-        public Workspace Workspace { get; private set; }
+        public Workspace? Workspace { get; private set; }
 
-        public event EventHandler WorkspaceChanged;
+        public event EventHandler? WorkspaceChanged;
 
         internal void SetWorkspaceAndRaiseEvents(Workspace workspace)
```

Comment:
```suggestion
        internal void SetWorkspaceAndRaiseEvents(Workspace? workspace)
```

Due to this:

https://github.com/dotnet/roslyn/blob/7daa213b210ba67d7eef65848cb27b0bfadda4de/src/Workspaces/Core/Portable/Workspace/Workspace_Registration.cs#L65

That other file might be tricky to annotate, but didn't want to leave this one "obviously wrong".